app.json

```diff
@@ -34,9 +34,6 @@
     "API_ACCESS_TOKEN": {
       "required": true
     },
-    "INVITE_LAMBDA_URL": {
```

Comment:
This is used for Heroku PR apps and would break them if it's removed (the default resolving behavior can't work for those)